tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "commonjs",
+    "allowJs": true,
+    "outDir": "dist",
+    "sourceMap": true
```

Comment:
For not using .json imports with declaration files you forgot to point in config file next configs
```
"esModuleInterop": true,
"moduleResolution": "node",
"resolveJsonModule": true
```
Please consider including them. Then you can remove `typings.d.ts` file ata all and call your .json config as
```
import config from './config/config.json'

console.log(config.name);
```